agent/scripts/newrelic.ini.template

```diff
@@ -638,10 +638,11 @@ newrelic.daemon.logfile = "/var/log/newrelic/newrelic-daemon.log"
 ;
 ;          Must be one of the following values:
 ;          cakephp, codeigniter, drupal, drupal8, joomla, kohana, laravel,
-;          magento, magento2, mediawiki, silex, slim, symfony1, symfony2,
+;          magento, magento2, mediawiki, slim, symfony2, symfony4,
 ;          wordpress, yii, zend, zend2, no_framework
 ;
-;          Note that "drupal" covers only Drupal 6 and 7.
+;          Note that "drupal" covers only Drupal 6 and 7 and "symfony2"
+;          now only supports Symfony 3.x.
```

Comment:
Maybe add a note that symfony4 covers Symfony 5 (see https://github.com/newrelic/newrelic-php-agent/pull/256)?

Reply:
I was more just emphasizing the "symfony2" doesnt actually support Symfony 2.x.

Reply:
I'm fine with merging it as is and I think [Zach's suggestion](https://github.com/newrelic/newrelic-php-agent/pull/579#discussion_r1024456730) makes more sense.